spec/system/assignment/index_spec.rb

```diff
@@ -54,98 +49,35 @@
         create :assignment, start_date: 3.days.ago, end_date: 3.days.since,
                             user: roster_user(roster), roster: roster
         visit roster_assignments_url(roster)
-        expect(page).to have_selector('td .cal-event.assignment', count: 7)
+        expect(page).to have_selector(
+          '.assignment-event:not(.assignment-event-owned)'
+        )
       end
     end
 
-    context 'start of assignment' do
-      it 'has a left radius' do
-        create :assignment, start_date: 3.days.ago, end_date: 3.days.since,
-                            user: user, roster: roster
+    context 'clicking on an empty day' do
+      it 'sends you to create a new assignment' do
         visit roster_assignments_url(roster)
-        expect(page).to have_selector('td .cal-event.assignment-start',
-                                      count: 1)
+        execute_script 'calendar.gotoDate("2021-02-01")'
```

Comment:
Similar to my previous comment, it might be a better approach to initiate this script execution with a click.

Reply:
Hmmm. I guess the way a "real person" would approach the task, "go to February, 2021" would be to:

1. visit the page
1. determine whether that month is in the past, now, or the future
2. if now, stop. if past, click "prev". if future, click "next".
3. keep clicking until the page has the text, "February, 2021"

Maybe it would be better just to click on the 14th of "this month", whatever that may be? I just hate introducing dependencies on "now" into tests, always seems to bite us.

Reply:
But, eh. I do it elsewhere in the spec....

Reply:
I think it's probably fine to not click as a user if you think it's trivial and it saves on complexity and test time. I'll leave the decision to you.